src/doc/using_visit/Quantitative/Expressions.rst

```diff
@@ -1967,15 +1967,16 @@ resrad Function: ``resrad()`` : ``resrad(expr0)``
 crack width Function: ``crack_width()`` : ``crack_width(crack_num, <crack1_dir>, <crack2_dir>, <crack3_dir>, <strain_tensor>, volume2(<mesh_name>))``
 
     Calculates crack width using the following formula:
-        crackwidth = L * (1 - (exp(-delta))  
-        where: 
-          L = ZoneVol / (Area perpendicular to crack_dir)
-            find Area by slicing the cell by plane with origin == cell center
-            and Normal == crack_dir.  Take area of that slice.
-    
-          delta = T11 for crack dir1 = component 0 of strain_tensor 
-                  T22 for crack dir2 = component 4 of strain_tensor
-                  T33 for crack dir3 = component 8 of strain_tensor
+        | crackwidth = L * (1 - (exp(-delta))  
```

Comment:
Trying to understand reStructuredText formatting here...why is the `|` char needed on each line here? If we're aiming for verbatim quote, doesn't a single `::` at end of 1969 work?

Reply:
FYI...if you go to the rendered manual for this PR, it looks like the function name isn't quite typeset right..

![Screen Shot 2022-11-04 at 2 39 13 PM](https://user-images.githubusercontent.com/5720676/200078969-67256aec-3ea7-4541-b331-9b114f6aebdb.png)

Reply:
Previously, building the manuals issued an 'unexpected indent' warning on line 1972.
I tried using the code block designation but received the same warning.
And the the indentation that I wanted to see wasn't preserved in either case.
Using the line block `|` character allows the indentation to be preserved.
You can take a look at the page as it currently exists and compare it with how it renders now in the PR.

Reply:
Hmm...odd...If you look earlier in that file for another similar case, I see one for `Ghost Zoneid Function` and it seems to render fine. Though, I see a stray single `:` following that block worth removing too.

That said, is this logic you are including here intended to give user a detailed understanding of what is being computed? If so, my suggestion is to follow what we do for several of the tensor functions. For example, look at doc source for `effective_tensor` function there. I think this is a much preferred way to go because it doesn't rely upon duplicating in docs what is already true in the code and yet gives any user so inclined sufficient details to understand what is happening...though you may need to improve the in-line comments for the relevant code block(s).

Reply:
I can fix the rendering of the function name by removing the blank line before the description.

Reply:
Inclusion of code block isn't feasible here. I can remove the explanation, but it was previously approved as-is. I was just trying to improve the indentation and remove the warning I noticed when building.

Reply:
Ok, lets keep what you changed then.